app/controllers/report_templates_controller.rb

```diff
@@ -0,0 +1,30 @@
+class ReportTemplatesController < TemplatesController
+  include Foreman::Controller::Parameters::ReportTemplate
+  helper_method :documentation_anchor
+
+  def documentation_anchor
+    '4.11ReportTemplates'
+  end
+
+  def generate
+    find_resource
+    headers["Cache-Control"] = "no-cache"
+    headers["Content-Disposition"] = %(attachment; filename="#{@template.suggested_report_name}")
+    safe_render(@template)
+  end
+
+  private
+
+  # not loading any vars, such as @host
+  def load_template_vars
+  end
+
+  def action_permission
+    case params[:action]
+      when 'generate'
+        'generate_report_templates'
```

Comment:
Same here, 'generate' is the right value.

Reply:
fixed